plugins/cpp_metrics/service/include/service/cppmetricsservice.h

```diff
@@ -0,0 +1,43 @@
+#ifndef CC_SERVICE_DUMMY_DUMMYSSERVICE_H
+#define CC_SERVICE_DUMMY_DUMMYSSERVICE_H
+
+#include <memory>
+#include <vector>
+
+#include <boost/program_options/variables_map.hpp>
+
+#include <odb/database.hxx>
+#include <util/odbtransaction.h>
+#include <webserver/servercontext.h>
+
+#include <CppMetricsService.h>
+
+namespace cc
+{
+namespace service
+{
+namespace cppmetrics
+{
+
+class CppMetricsServiceHandler : virtual public CppMetricsServiceIf
```

Comment:
The service seems to be incomplete and copy-paste from the *dummy* plugin, so I will not review that further yet.

Reply:
There is still a `void getDummyString(std::string& str_)` method in `CppMetricsServiceHandler`. While the service is not implemented yet for this plugin, this dummy code should still be removed in my view.

Reply:
Fixed in 667361582fcf5b58c66ed660fe37a2aa57b5fa59.